src/Core/Configurations/JsonConfigSchemaValidator.cs

```diff
@@ -71,8 +85,10 @@ public async Task<JsonSchemaValidationResult> ValidateJsonConfigWithSchemaAsync(
         {
             try
             {
-                JsonSchema jsonSchema = await JsonSchema.FromUrlAsync(runtimeConfig.Schema);
-                return jsonSchema.ToJson();
+                // Send a GET request to the URL specified in runtimeConfig.Schema to get the JSON schema.
+                HttpResponseMessage response = await _httpClient.GetAsync(runtimeConfig.Schema);
```

Comment:
This restricts us to using a file hosted behind a url. Is there any way to support a user providing their own local file to use for schema validation? e.g. environment with no internet connection or our own local testing of dab: we don't have a valid url pointing to a JsonSchema (`https://github.com/Azure/data-api-builder/releases/download/vmajor.minor.patch/dab.draft.schema.json`) until an official release is published on github and the pipeline substitutes the major/minor/patch values.

So in addition to `dab -validate dab-config.json`, the user can also do `dab -validate dab-config.json C:\files\dab-config-schema.json`

Reply:
allowing us to use a local file can be separate work (open an issue and respond with the issue number here to close out)

Reply:
in case the fetch from url fails we get the jsonSchema from the package itself. So we always have schema to validate from